AdornUI-Stylesheets/image.css

```diff
@@ -0,0 +1,8 @@
+.img-responsive {
+  width: 100%;
+  height: auto;
+}
+.img-round {
+  border-radius: 50%;
+  aspect-ratio: 1/1;
```

Comment:
Hey - didn't know about `aspect-ratio` - does this mean rectangular images will also become perfect circle with border radius 50%?